MonstieWash/Assets/Scripts/UI/ScoreUIManager.cs

```diff
@@ -0,0 +1,137 @@
+{
+    public string name;
+    public bool complete;
+    public int score;
+
+    public PHTask(string _name, bool _complete, int _score)
+    {  name = _name; complete = _complete; score = _score; }
+}
+
+public class ScoreUIManager : MonoBehaviour
+{
+    [SerializeField] private Image m_clipboard;
+    [SerializeField] private GameObject m_lineItem;
+    [SerializeField] private TextMeshProUGUI m_totalScore;
+    [SerializeField] private GameObject m_strikeOut;
+
+    //Probably temporary, may change to calc scale with number of tasks later.
+    [SerializeField][Range(0.0f, 5.0f)] private float m_spacing;
+
+    /// <summary>
+    /// Placeholder. Suggest Json to player prefs.
+    /// </summary>
+    [SerializeField] private List<PHTask> m_tasks;
+    [SerializeField] private List<GameObject> m_LIList;
+
+    [SerializeField][Range(0.0f, 2.0f)] private float m_secondsWait = 1.0f;
+    private bool m_skip = false;
+
+    // Start is called before the first frame update
+    private void OnEnable()
+    {
+        MenuInputManager.Inputs.OnAltSelect += Inputs_OnAlt_Select;
+
+        m_tasks = PlaceholderTaskFill();
+
+        StartCoroutine(LineItemSetActive());
+    }
+
+    private void OnDisable()
+    {
+        MenuInputManager.Inputs.OnAltSelect -= Inputs_OnAlt_Select;
+    }
+
+    /// <summary>
+    /// Fills the task list and returns it.
+    /// </summary>
+    /// <returns>List of tasks with score multiplier</returns>
+    private List<PHTask> PlaceholderTaskFill()
+    {
+        List<PHTask> ph_tasks = new List<PHTask>();
+
+        for (int i = 0; i < 8; i++)
+        {
+            ph_tasks.Add(new PHTask("Task Name Here", true, 10));
+        }
+
+        return ph_tasks;
+    }
+
+    private IEnumerator LineItemSetActive()
+    {
+        Transform clipTransform = m_clipboard.transform;
+        RectTransform LITransform;
+
+        TextMeshProUGUI name = null;
+        TextMeshProUGUI score = null;
+        GameObject strikeOut = null;
+
+        int totalScore = 0;
+
+        for (int i = 0; i < m_tasks.Count; i++)
+        {
+            m_LIList.Add(Instantiate(m_lineItem, clipTransform));
+            m_LIList[i].SetActive(false);
+            LITransform = m_LIList[i].GetComponent<RectTransform>();
+            LITransform.anchoredPosition = new Vector2(0, LITransform.rect.height / m_spacing * -i);
+
+            name = m_LIList[i].transform.Find("TaskName").GetComponent<TextMeshProUGUI>();
+            strikeOut = m_LIList[i].transform.Find("StrikeOut").gameObject;
+
+            name.text = m_tasks[i].name;
+
+            m_LIList[i].SetActive(true);
+            strikeOut.SetActive(false);
+        }
+
+        for (int j = 0; j < m_LIList.Count; j++)
+        {
+            for (float timer = m_secondsWait; timer >= 0; timer -= Time.deltaTime)
+            {
+                if (m_skip)
+                {
+                    break;
+                }
+                yield return null;
+            }
```

Comment:
I have honestly never in all my days seen a for loop used like this but I mean if it works it works I suppose. The only thing is just to try make sure to use float values when working with them.
```suggestion
            for (float timer = m_secondsWait; timer >= 0f; timer -= Time.deltaTime)
            {
                if (m_skip)
                {
                    break;
                }
                yield return null;
            }
```